clang-tools-extra/docs/clang-tidy/checks/bugprone/unsafe-crtp.rst

```diff
@@ -0,0 +1,62 @@
+.. title:: clang-tidy - bugprone-unsafe-crtp
+
+bugprone-unsafe-crtp
+====================
+
+Finds CRTP used in an error-prone way.
+
+If the constructor of a class intended to be used in a CRTP is public, then
+it allows users to construct that class on its own.
+
+Example:
```

Comment:
(Not sure this is needed. The example is rendered differently because it is code text.)

Reply:
This pattern is used by multiple other check documentations too. I'm just being consistent here.